### PR TITLE
Support new operations in TS: Selu, Swish, HSwish, Tile, CumSum, HardSigmoid

### PR DESCRIPTION
**Pull request by evkotov**
### Details:
 - add support of Selu, Swish, Tile, CumSum, HardSigmoid, LogSoftmax, ConvertLike to TransposeSinking
 - add unit tests

### Tickets:
 - 120407

### CODE REVIEW

File: src/common/transformations/src/transformations/transpose_sinking/ts_general.cpp [hunk 1]
```diff
@@ -58,6 +62,8 @@ TSGeneralBackward::TSGeneralBackward() {
     add_matcher<TSInterpolateBackward>();
     add_matcher<TSSliceBackward>();
     add_matcher<TSGatherBackward>();
+    add_matcher<TSCumSumBackward>();
+    add_matcher<TSTileBackward>();
```
Comment (ilya-lavrenov) [inline src/common/transformations/src/transformations/transpose_sinking/ts_general.cpp:66]: I suppose we should use `ADD_MATCHER` macro to enable CC in this umbrella transformation

**Review by itikhono — APPROVED** [review]

**Comment by itikhono** [comment]
it's better to run performance validation for tf and tf light models

**Comment by itikhono** [comment]
waiting for TF hub tests in https://github.com/openvinotoolkit/openvino/pull/20062
let's double check nncf calibration after the merge, most probably it's not affected

**Comment by itikhono** [comment]
nightly TF Hub models passed without new issues: 4 models failed due to timeout
This is expected

**Pull request merged**